internal/cli/atlas/cloudprovider/aws/create.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package aws
+
+import (
+	"github.com/mongodb/mongocli/internal/cli"
+	"github.com/mongodb/mongocli/internal/config"
+	"github.com/mongodb/mongocli/internal/flag"
+	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/spf13/cobra"
+)
+
+const (
+	provider       = "AWS"
+	createTemplate = "AWS IAM role successfully created.\n"
```

Comment:
[nit] you need to print the AWS info here as it's needed by the users for the next steps, in particular
`atlasAWSAccountArn`, `atlasAssumedRoleExternalId`,  `roleId`, you may want to do something similar to what we did for API credentials that show multiple lines of data here